deployment/scripts/neo4j/customer-os.cypher

```diff
@@ -36,24 +36,24 @@ MATCH (t:Tenant {name:"openline"})
 MATCH (t:Tenant {name:"openline"})
     MERGE (u:User {id:"development@openline.ai"})-[:USER_BELONGS_TO_TENANT]->(t)
     ON CREATE SET
-    		u.firstName ="Development",
+    		u.firstName="Development",
             u.lastName="User",
     		u.createdAt=datetime({timezone: 'UTC'}),
     		u.source="openline",
     		u.sourceOfTruth="openline",
     		u.appSource="manual";
 
 MATCH (u:User {id:"development@openline.ai"})
```

Comment:
Change to avoid issues when same email is in multiple tenants.
MATCH (u:User {id:"development@openline.ai"})-[:USER_BELONGS_TO_TENANT]->(t:Tenant {name:"openline"})

Reply:
fixed